chalicelib/checks/wfr_checks.py

```diff
@@ -204,6 +204,41 @@ def md5run_status(connection, **kwargs):
     return check
 
 
+@check_function()
+def md5run_released_files(connection, **kwargs):
+    """Search for (pre)released files that do not have md5sum (this should not happen)"""
+    check = CheckResult(connection, 'md5run_released_files')
+    my_auth = connection.ff_keys
+    check.action = "md5run_start"
+    check.status = 'PASS'
+    # check indexing queue
+    check, skip = wfr_utils.check_indexing(check, connection)
+    if skip:
+        return check
+    # Build the query
+    statuses = ['pre-release', 'released',  # 'released to project', 'archived to project'
+                'archived', 'replaced']
```

Comment:
I know why you are not currently including '... to project' files.  What is the reason for not including uploaded files in the check?  When things are working correctly is not the status change done after the md5 has been run so uploaded files should always have an md5sum?  Just curious about the current state of things and if my understanding on what should happen is correct.

Reply:
You are right: for most files, the status is changed to uploaded only after md5sum is patched, so this would be already a good time to check if anything went wrong. ENCODE pipelines are an exception, since they won't have md5sum, but it would be good to check already when uploaded.
I just had to go back and patch also the uploaded files currently missing md5.